### ISSUE TITLE
Feature request - Shard.git_branch

### ISSUE THREAD

**Issue by wwalker** [maiha/shard.cr#1]
It would be very nice to have `Shard.git_branch`

**Comment by maiha** [comment]
@wwalker 
Right!
Just released `v1.1.0` that would contain what you want :)
Enjoy Crystal!

```console
$ cat test.cr 
require "./src/shard"
p Shard.git_branch

$ crystal test.cr
"main"

$ git checkout -b foo
Switched to a new branch 'foo'

$ crystal test.cr
"foo"
```

**Comment by waywalke** [comment]
Thank you!  I call this fixed.
